cups/ppd-cache.c

```diff
@@ -1364,6 +1364,30 @@ _ppdCacheCreateWithPPD(ppd_file_t *ppd)	/* I - PPD file */
 
   if ((media_type = ppdFindOption(ppd, "MediaType")) != NULL)
   {
+    static const struct {
+      const char *ppd_name;
+      size_t ppd_name_match_length;
```

Comment:
Descriptive name but long - maybe just `match_length`? Also, please indent start of name to match the other members...

Reply:
Both done in the latest update.